libcaf_core/tests/legacy/mixin/requester.cpp

```diff
@@ -363,4 +363,23 @@ TEST_CASE("GH-1299 regression blocking") {
   }
 }
 
+TEST_CASE("GH-698 regression responding to a request drops the strong "
+          "reference count") {
```

Comment:
```suggestion
TEST_CASE("actors cancel request timeouts after receiving a response") {
```

Reply:
All right, since it's a quite subtle issue, let's just name the test `TEST_CASE("GH-698 regression")`.